app/home/home.component.spec.ts

```diff
@@ -0,0 +1,41 @@
+import {provide} from 'angular2/core';
+import {it, describe, expect, injectAsync, beforeEachProviders} from 'angular2/testing';
+import {TestComponentBuilder, ComponentFixture, setBaseTestProviders} from 'angular2/testing';
```

Comment:
I'm fine with the aesthetics, maybe we can add our own test file to include that makes this and other imports easier?
Looks like lines 1-7 will be pretty common in many tests.